### PR TITLE
Task changes

### PR DESCRIPTION
**Pull request by aravindavk**

### CODE REVIEW

**Comment by aravindavk** [comment]
WIP: Some major refactoring. Lot of things now simplified. Will complete refactoring by this week.

File: server/src/db/brick.cr [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    bricks.map do |brick|
+      brk = MoanaTypes::Brick.new
+      brk.id = brick.id
+      brk.path = brick.path
+      brk.port = brick.port
+      brk.device = brick.device
+      brk.state = brick.state
+      brk.type = brick.type
+
+      brick
```
Comment (amarts) [inline server/src/db/brick.cr]: `brk`?

Comment (aravindavk) [reply server/src/db/brick.cr]: ack

**Comment by aravindavk** [comment]
Commits are properly organized in https://github.com/kadalu/moana/pull/9

Closing this.

**Pull request closed without merging**